files/en-us/web/api/serial/onconnect/index.html

```diff
@@ -0,0 +1,51 @@
+---
+title: Serial.onconnect
+slug: Web/API/Serial/onconnect
+tags:
+  - API
+  - Property
+  - Reference
+  - Serial.onconnect
+  - Serial
+---
+<div>{{securecontext_header}}{{DefaultAPISidebar("Serial API")}}</div>
+
+<p class="summary">The <strong><code>onconnect</code></strong> {{domxref("EventHandler")}} of the {{domxref("Serial")}} interface is called when a port has been disconnected from the device. This method receives an {{domxref("Event")}} object. The target of this event is the {{domxref("SerialPort")}} interface that has been disconnected.
```

Comment:
```suggestion
<p class="summary">The <strong><code>onconnect</code></strong> {{domxref("EventHandler")}} of the {{domxref("Serial")}} interface is called when a port has been connected from a device. This method receives an {{domxref("Event")}} object. The target of this event is the {{domxref("SerialPort")}} interface that has been disconnected.
```

Do you mean connected not disconnected?